hack/validate-ci-operator-config-filename.py

```diff
@@ -23,29 +23,24 @@ def main():
     errors = []
 
     if not os.path.isdir(args.config_dir):
-        errors.append(
-            "[ERROR] {} directory doesn't exist.".format(args.config_dir))
+        errors.append(f"[ERROR] {args.config_dir} directory doesn't exist.")
```

Comment:
Wow, I've never seen this before. Leave it to Python to invent another way to format strings...

Reply:
Hehe. Yeah, this is 3.6+ stuff, but I have checked we are using 3.7 to run this, so I thought we might use it, it's nicely readable.